builders/interact/proxy-accounts.md

```diff
@@ -0,0 +1,181 @@
+
+Proxy accounts can be set up to perform specific Substrate functions such as author mapping, staking, balances, and more. This can allow you to, for example, grant a trusted individual access to perform collator or delegator functions on your behalf. A proxy could also be used to keep a staking account safe in cold storage.
+
+This guide will show you how to set up a proxy account on the Moonbase Alpha TestNet for balance transfers and how to execute a proxy transaction.
+
+## Checking Prerequisites
+
+To follow along with this tutorial, you will need to have:
+
+- [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/explorer){target=_blank} open and connected to Moonbase Alpha
+- Create or have two accounts on Moonbase Alpha
+- At least one of the accounts will need to be funded with `DEV` tokens. You can obtain tokens for testing purposes from the Moonbase Alpha [faucet](/builders/get-started/moonbase/#get-tokens/)
+
+If you need help importing your accounts into Polkadot.js Apps, please check out the [Interacting with Moonbeam using Polkadot.js Apps](/tokens/connect/polkadotjs/#creating-or-importing-an-h160-account) guide.
+
+## General Definitions
+
+When setting up a proxy account, a bond for the proxy will be taken out of your free balance and moved to your reserved balance. The bond is required as adding a proxy requires on-chain storage space. After the proxy has been removed from your account, the bond will be returned to your free balance.
+
+The deposit is calculated based on a deposit base and a deposit factor:
+
+- **Deposit base** - the amount to be reserved for an account to have a proxy list
+- **Deposit factor** - the additional amount to be reserved for every proxy the primary account has
+
+The equation for calculating the deposit is: `deposit base + deposit factor * number of proxies`.
+
+=== "Moonriver"
+    |    Variable    |                       Value                        |
+    |:--------------:|:--------------------------------------------------:|
+    |  Deposit base  |  {{ networks.moonriver.proxy.deposit_base }} MOVR  |
+    | Deposit factor | {{ networks.moonriver.proxy.deposit_factor }} MOVR |
+    |  Max proxies   | {{ networks.moonriver.proxy.max_proxies }} proxies |
+
+=== "Moonbase Alpha"
+    |    Variable    |                       Value                        |
+    |:--------------:|:--------------------------------------------------:|
+    |  Deposit base  |   {{ networks.moonbase.proxy.deposit_base }} DEV   |
+    | Deposit factor |  {{ networks.moonbase.proxy.deposit_factor }} DEV  |
+    |  Max proxies   |  {{ networks.moonbase.proxy.max_proxies }} proxies |
+
+## Proxy Types
+
+When creating a proxy account, you must choose a type of proxy which will define how the proxy can be used. The available options are:
+
+- **`AuthorMapping`** - this type of proxy account is allowed to make transactions related to governance
```

Comment:
this type of proxy is used by collators to migrate services from one server to another